test/inductor/test_compiled_autograd.py

```diff
@@ -3638,6 +3653,71 @@ def wrap_test_class(orig_cls):
         test_dtensor.TestDTensorCompile
     )
 
+
+class TestCompiledAutogradOpInfo(TestCase):
+    def reset(self) -> None:
+        torch._logging.set_logs(compiled_autograd_verbose=False)
+        config.compiled_autograd = False
+        compiled_autograd.reset()
+
+    @ops(hop_db, allowed_dtypes=(torch.float,))
+    def test_hops_in_bwd(self, device, dtype, op):
+        if self.id() in known_failing_tests:
+            self.skipTest("Expected to fail.")
+
+        self.reset()
```

Comment:
nit: we could put this in a setUp and tearDown, that's the idiomatic way to do it